src/EnergyPlus/ChillerIndirectAbsorption.cc

```diff
@@ -516,15 +516,25 @@ namespace ChillerIndirectAbsorption {
 			IndirectAbsorber( AbsorberNum ).TempDesCondIn = rNumericArgs( 6 );
 			IndirectAbsorber( AbsorberNum ).MinCondInletTemp = rNumericArgs( 7 );
 			IndirectAbsorber( AbsorberNum ).TempLowLimitEvapOut = rNumericArgs( 8 );
-			IndirectAbsorber( AbsorberNum ).EvapVolFlowRate = rNumericArgs( 9 );
-			if ( IndirectAbsorber( AbsorberNum ).EvapVolFlowRate == AutoSize ) {
+			if ( rNumericArgs( 9 ) != 0.0 ){
```

Comment:
So now that we have `\default autosize`, if it is blank, it will be changed to `AutoSize` by the input processor.  Does that mean we can simplify this down a little?  It will never be zero now.  It will be either:
 - blank which means -99999
 - "AutoSize" which means -99999, or
 - greater than zero.  

So this block: 

```
if ( rNumericArgs( 9 ) != 0.0 ){
  IndirectAbsorber( AbsorberNum ).EvapVolFlowRate = rNumericArgs( 9 );
	if ( IndirectAbsorber( AbsorberNum ).EvapVolFlowRate == AutoSize ) {
	    IndirectAbsorber( AbsorberNum ).EvapVolFlowRateWasAutoSized = true;
	}
} else {
	IndirectAbsorber( AbsorberNum ).EvapVolFlowRate = AutoSize;
        IndirectAbsorber( AbsorberNum ).EvapVolFlowRateWasAutoSized = true;
}
```

should just be:

```
IndirectAbsorber( AbsorberNum ).EvapVolFlowRate = rNumericArgs( 9 );
if ( IndirectAbsorber( AbsorberNum ).EvapVolFlowRate == AutoSize ) {
	IndirectAbsorber( AbsorberNum ).EvapVolFlowRateWasAutoSized = true;
}
```